### PR TITLE
Add the VS Code home page samples

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the VS Code home page samples.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by keizer619** [comment]
There are `.DS_Store` committed

File: vs-code-home-page-samples/data-mapper/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[package]
+org = "praneesha"
```
Comment (keizer619) [inline vs-code-home-page-samples/data-mapper/Ballerina.toml]: Lets change org to `integration_samples`. Need to fix in other places as well

File: vs-code-home-page-samples/http-api-designer/main.bal [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    resource function head [string path](@http:Header string x\-greeting, string name) returns string {
+        return x\-greeting + " from HEAD " + path + " to " + name;
+    }
+
+    resource function options [string path](@http:Header string x\-greeting, string name) returns string {
+        return x\-greeting + " from OPTIONS " + path + " to " + name;
+    }
+}
```
Comment (keizer619) [inline vs-code-home-page-samples/http-api-designer/main.bal:61]:
```suggestion
}

```

Comment (keizer619) [inline vs-code-home-page-samples/http-api-designer/main.bal]: Lets add a new line.

Comment (praneesha) [reply vs-code-home-page-samples/http-api-designer/main.bal]: Fixed.

File: vs-code-home-page-samples/data-mapper/Ballerina.toml (hunk 1 above)
Comment (praneesha) [reply vs-code-home-page-samples/data-mapper/Ballerina.toml]: Fixed.

**Review by hevayo — APPROVED** [review]

**Pull request merged**